packages/framework/src/Model/Localization/Exception/AdministrationLocaleNotFoundException.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Model\Localization\Exception;
+
+use Exception;
+use RuntimeException;
+
+class AdministrationLocaleNotFoundException extends RuntimeException implements LocalizationException
+{
+    /**
+     * @param string $adminLocale
+     * @param array $possibleLocales
+     * @param \Exception|null $previous
+     */
+    public function __construct(string $adminLocale, array $possibleLocales, Exception $previous = null)
+    {
+        $message = sprintf(
+            'You tried to use administration in "%s" locale, but you have registered only ["%s"].'
+            . ' Either register "%s" as a locale with some domain or use one of ["%s"] as administration locale.',
+            $adminLocale,
+            implode('","', $possibleLocales),
+            $adminLocale,
+            implode('","', $possibleLocales)
```

Comment:
You can avoid the duplication of parameters using argument swapping (see [docs](http://php.net/manual/en/function.sprintf.php)):
```php
sprintf(
  `You tried to use administration in "%1$s" locale, but you have registered only ["%2$s"]. Either register "%1$s" as a locale with some domain or use one of ["%2$s"] as administration locale.',
  $adminLocale,
  implode('","', $possibleLocales)
)
```

Reply:
Oh, that's nice. Thanks